stable_baselines3/ppo/ppo.py

```diff
@@ -146,6 +148,7 @@ def __init__(
         self.n_epochs = n_epochs
         self.clip_range = clip_range
         self.clip_range_vf = clip_range_vf
+        self.normalize_advantage = normalize_advantage
```

Comment:
if I recall, there is a check above that is valid only if normalize advantage is True

Reply:
Hey sorry I don't quite get this...

Reply:
read the lines above, there is a check that is no longer valid when `normalize_advantage=False` but this is minor

Reply:
Are you referring to the lines here: https://github.com/DLR-RM/stable-baselines3/blob/3084b81182f956149fb9235237f1e78f028914f1/stable_baselines3/ppo/ppo.py#L125-L146? They do not seem to relate to `batch_size` and I don't see how `normalize_advantage=False` will make those checks invalid.

Reply:
https://github.com/DLR-RM/stable-baselines3/issues/440, `batch_size=1` is ok when no normalization is used as mentioned in the comment ;)

Reply:
but could you also add a "running" test, that just check we can run PPO with `normalize_advantage=False` ?

Reply:
> https://github.com/DLR-RM/stable-baselines3/issues/440, batch_size=1 is ok when no normalization is used as mentioned in the comment ;)

Oh I see 😅, definitely did not think of that. Just want to make sure I understand, do you want to support `batch_size=1` if `normalize_advantage=False`?

Reply:
yes if it is easy to have, no otherwise ;)